### PR TITLE
Multiple tweaks

### PR DESCRIPTION
**Pull request by aspacca**
Cleanup

## What does this PR do?
ES client:
- sets max retries to 4: max value according to `initial_backoff * 2 ** retry_number` with `initial_backoff = 2` and max backoff 600 secs
- enables http compress by default

Internal renaming of `trigger_type` for replay queue to `replay-sqs`

## Why is it important?
Cleaner code

## Checklist

- [x] My code follows the style guidelines of this project
~- [ ] I have commented my code, particularly in hard-to-understand areas~
~- [ ] I have made corresponding changes to the documentation~
~- [ ] I have made corresponding change to the default configuration files~
~- [ ] I have added tests that prove my fix is effective or that my feature works~
- [x] I have added an entry in `CHANGELOG.md`

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+### v0.20.1 - 2022/02/08
+##### Bug fixes
+* HTTP compression always on and max retries to not exceed 15 mins in the ES client [#69](https://github.com/elastic/elastic-serverless-forwarder/pull/69)
```
Comment (masci) [inline CHANGELOG.md:3]: I feel like a verb is missing, maybe you meant `Set HTTP compression...`?

**Review by endorama — APPROVED** [review]
One question, but is not a blocker: why limiting retry to 15 minutes? Is this mandated by something on ES side?

**Comment by aspacca** [comment]
> One question, but is not a blocker: why limiting retry to 15 minutes? Is this mandated by something on ES side?

the lambda cannot run more than 15 minutes. assuming a single bulk request will keep failing we should still be able to return before the lambda times out so that it can execute the error handling code

File: CHANGELOG.md (hunk 1 above)
Comment (aspacca) [reply CHANGELOG.md:3]: indeed: explicit is better than implicit :)

**Pull request merged**